tracer/src/Datadog.Trace/Iast/StackWalker.cs

```diff
@@ -83,22 +83,22 @@ public static bool AssemblyExcluded(string assembly)
         return excluded;
     }
 
-    // For performance reasons, we are not supporting wildcards fully. We just need to use '*' at the end for now. We can use regular expressions
+    // For performance reasons, we are not supporting wildcards fully. We just need to use '.' at the end for now. We can use regular expressions
     // if in the future we need a more sophisticated wildcard support
     private static bool IsExcluded(string assembly)
     {
         foreach (var assemblyToSkip in AssemblyNamesToSkip)
         {
-            if (assemblyToSkip.EndsWith("*"))
+            if (assemblyToSkip.EndsWith("."))
```

Comment:
nit: tiny optimization, but checking for a `char` does a lot less work than for a string 🙂 
```suggestion
            if (assemblyToSkip.EndsWith('.;))
```

Reply:
It is not supported under netstandard or net framework, but for the other cases, it will improve the performance. Thank you!